website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -0,0 +1,627 @@
+
+
+![Metrics view image](/img/react-ex-metrics.png)
+
+
+Our metrics are great for understanding user traffic. You can get a better sense of:
+
+
+* What time(s) of the day or week are requests the highest?
+* Which feature flags are the most popular?
+
+
+Another use case for reviewing metrics is verifying that the right users are being exposed to your feature based on how you’ve configured your strategies and/or variants.
+
+Take a look at our [Metrics API documentation](/reference/api/unleash/metrics) to understand how it works from a code perspective.
+
+
+## Feature Flag Audit Logs in React
+
+Because a Feature Flag service controls the way an application behaves in production, it can be highly important to have visibility into when changes have been made and by whom. This is especially true in highly regulated environments, such as health care, insurance, banking, and others. In these cases (or similar), you might find audit logging useful for:
+
+
+1. Organizational compliance
+2. Change control
+
+Fortunately, this is straightforward in Unleash Enterprise.
+
+Unleash provides the data to log any change that has happened over time, at the flag level from a global level. In conjunction with Unleash, tools like [Splunk](https://www.splunk.com/) can help you combine logs and run advanced queries against them. Logs are useful for downstream data warehouses or data lakes.
+
+In our React tutorial application, we can [view Event logs](http://localhost:4242/projects/default/features/newNotificationsBadge/logs) to monitor the changes to flag strategies and statuses we have made throughout our examples, such as:
+
+
+* When the newNotificationsBadge flag was created
+* How the gradual rollout strategy was configured
+* When and how the variants were created and configured
+
+
+![Events log image](/img/react-ex-event-log.png)
+
+
+You can also retrieve event log data by using an API command below:
+
+```
+curl -L -X GET '<your-unleash-url>/api/admin/events/:featureName' \
+-H 'Accept: application/json' \
+-H 'Authorization: <API_KEY_VALUE>'
+```
+
+Read our documentation on [Event logs](/reference/event-log) and [APIs](/reference/api/unleash/events) to learn more.
+
+
+## Change Management & Feature Flag Approvals in React
+
+Unleash makes it easy to toggle a feature. But the more users you have, the more risk with unexpected changes occurring. That’s why we implemented an approval workflow within Unleash Enterprise for making a change to a feature flag. This functions like a Git pull request, and models a Git review workflow. You could have 1 or more approvals required to reduce risk of someone changing something they shouldn’t. It helps development teams to have access only to what they _need_. For example, you can use Unleash to track changes to your React feature flag’s configuration.
```

Comment:
Pull requests are a GitHub thing, not a Git thing.

Also: I don't think we have a standard of when we switch from writing numbers out in letters to using numerals, but I'd probably use 'one' instead of '1' here. But that's up to you ☺️ 

Finally, that last sentence feels like a bit of a non-sequitur. Is that intended? Why are we talking about tracking changes all of a sudden?

```suggestion
Unleash makes it easy to toggle a feature. But the more users you have, the more risk with unexpected changes occurring. That’s why we implemented an approval workflow within Unleash Enterprise for making a change to a feature flag. This functions similar to GitHub's pull requests, and models a Git review workflow. You could have one or more approvals required to reduce risk of someone changing something they shouldn’t. It helps development teams to have access only to what they _need_. For example, you can use Unleash to track changes to your React feature flag’s configuration.
```